.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+jobs:
+  java:
+    machine:
+      image: circleci/classic:latest
+    working_directory: ~/misk
+    steps:
+      - checkout # check out source code to working directory
+      - run:
+          name: "Download and extract OpenJDK 11"
```

Comment:
Is this because their `openjdk-11` isn't sufficient? Is there another docker image we could use instead of bootstrapping this ourselves? Seems like it will be very fragile and prone to require updating.

Reply:
The tests don't run in docker, they run in a VM. For various reasons it's difficult to get our tests to work outside of docker. Circle CI does not offer an openjdk 11 image for the VM.

This is not fragile. It is the official mirror for OpenJDK. We will just bump the version when we want to bump our Java version.